<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentClass</class>
 <widget class="QMainWindow" name="ExperimentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Experiment</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>图像采集</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Load">
        <property name="text">
         <string>Load</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Capture">
        <property name="text">
         <string>Capture</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>401</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>图像预处理</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>311</width>
       <height>712</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowminimumheight="50,50,50,50,50,50,50,50,50,50,50">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="grayImage">
        <property name="text">
         <string>图像灰度化</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QPushButton" name="Robert">
        <property name="text">
         <string>Robert
边缘检测</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QPushButton" name="Hist">
        <property name="text">
         <string>灰度直方图</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QPushButton" name="Soble">
        <property name="text">
         <string>Soble
边缘检测</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="equalizeHist">
        <property name="text">
         <string>直方图均衡化</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="equalizeHistCV">
        <property name="text">
         <string>直方图均衡化
OpenCV</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="GradientSharp">
        <property name="text">
         <string>梯度锐化</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="Gauss">
        <property name="text">
         <string>高斯噪声</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="Laplace">
        <property name="text">
         <string>Laplace锐化</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="LinearFilter">
        <property name="text">
         <string>线性滤波</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="SaltAndPepper">
        <property name="text">
         <string>椒盐噪声</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="MorphologicalFilter">
        <property name="text">
         <string>形态学滤波</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="Monocular">
        <property name="text">
         <string>单目标定</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="EdgeFilter">
        <property name="text">
         <string>保边均值滤波</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="Binocular">
        <property name="text">
         <string>双目标定</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="AffineTrans">
        <property name="text">
         <string>仿射变换</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="Stereo">
        <property name="text">
         <string>立体匹配</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="PerspectiveTrans">
        <property name="text">
         <string>透视变换</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="Canny">
        <property name="text">
         <string>Canny
边缘检测</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>目标检测</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>451</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="ThresholdSeg">
        <property name="text">
         <string>阈值分割</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="SIFT">
        <property name="text">
         <string>SIFT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="haar">
        <property name="text">
         <string>face-haar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="OTSU">
        <property name="text">
         <string>OTSU</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Brisk">
        <property name="text">
         <string>Brisk</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="ssd">
        <property name="text">
         <string>face-ssd</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="Kittle">
        <property name="text">
         <string>Kittle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="Brisk1">
        <property name="text">
         <string>Brisk-1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="yolo">
        <property name="text">
         <string>yolo</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="ThresholdSegCV">
        <property name="text">
         <string>阈值分割CV</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="ORB">
        <property name="text">
         <string>ORB</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="svm">
        <property name="text">
         <string>svm</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="InterFrameDiff">
        <property name="text">
         <string>帧间差分</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="ORB1">
        <property name="text">
         <string>ORB-1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="carcascade">
        <property name="text">
         <string>car-cascade</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="MixedGauss">
        <property name="text">
         <string>混合高斯模型</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="num">
        <property name="text">
         <string>svm-num</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="MixedGaussVideo">
        <property name="text">
         <string>混合高斯模型
video</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="tran">
        <property name="text">
         <string>svm-num-tran</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="singleGauss">
        <property name="text">
         <string>单高斯模型</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>目标跟踪</string>
    </property>
    <widget class="QPushButton" name="camshift">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Camshift</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
